doc/whats_new/v1.3.rst

```diff
@@ -147,6 +147,17 @@ Changelog
   inconsistent with the sckit-learn verion the estimator was pickled with.
   :pr:`25297` by `Thomas Fan`_.
 
+:mod:`sklearn.metrics`
+......................
+
+- |Enhancement| :class:`metrics.silhouette_samples` nows accepts a sparse
```

Comment:
Can this entry be moved down to the `sklearn.metrics` section below?

Reply:
Yes. I must have missed that when resolving the merge conflicts.